packages/related-keyphrase-suggestions/src/components/CountrySelector/index.js

```diff
@@ -0,0 +1,193 @@
+import React, { useCallback } from "react";
+import PropTypes from "prop-types";
+import { __ } from "@wordpress/i18n";
+import { Button, AutocompleteField } from "@yoast/ui-library";
+import classNames from "classnames";
+
+/**
+ * List of all available database countries for the Semrush API.
+@link https://www.semrush.com/api-analytics/#databases
+@type {{value: string, label: string}[]}
+ */
+const COUNTRIES = [
```

Comment:
This means that the country selector is not going to be a translatable thing, right?

Granted, it's a big thing to solve (also depending on previous [discussions](https://github.com/Yoast/reserved-tasks/issues/298#issuecomment-2449907165)/[tasks](https://github.com/Yoast/ai-frontend/issues/107))), but if that's the case we should create a task to track this.